.env.example

```
@@ -1,6 +1,6 @@
 export issuer=urn:gov:gsa:SAML:2.0.profiles:sp:sso:localhost
 export assertion_consumer_service_url=http://localhost:4567/consume
-export idp_sso_target_url=http://localhost:3000/api/saml/auth2018
-export idp_slo_target_url=http://localhost:3000/api/saml/logout2018
+export idp_sso_target_url=http://localhost:3000/api/saml/auth2021
+export idp_slo_target_url=http://localhost:3000/api/saml/logout2021
```

what if we made this dynamic?
```suggestion
export idp_sso_target_url=http://localhost:3000/api/saml/auth$(date +"%Y")
export idp_slo_target_url=http://localhost:3000/api/saml/logout$(date +"%Y")
```

I just verified with an example Dotenv that it works

---

ok so like maybe a `${YEAR:-date +"%Y"}` kinda thing

---

Ah, this is the `.env` - I'm good with your suggestion. I think I'm looking to implement a different option the user has (IAL, AAL, Encrypted, and new Endpoint) that would let me use the deployed sample app to test the different endpoints. But that should be a separate ticket so I'll go away now 😉

---

>what if we made this dynamic?

Huh, very cool! That definitely helps save some future maintenance burden. 👍 

> I think I'm looking to implement a different option the user has (IAL, AAL, Encrypted, and new Endpoint) that would let me use the deployed sample app to test the different endpoints.

That does seem like it'd be useful, FWIW.

But yeah, in my case, it was more a: "This project doesn't work from a fresh clone" kind of issue 😄